packages/compass/package.json

```diff
@@ -169,6 +169,7 @@
     "email": "compass@mongodb.com"
   },
   "dependencies": {
+    "@aws-sdk/credential-providers": "^3.348.0",
```

Comment:
We'd probably need in in the depcheck ignore as this is not used directly?

Reply:
Somehow it works, is a peer dep of `mongodb-client-encryption`